experimental/InvariantTheory/src/InvariantTheory.jl

```diff
@@ -484,7 +529,7 @@ function inv_generators(I::MPolyIdeal, G::LinearlyReductiveGroup, ringg::MPolyRi
     new_det = mapp_(det_)
 
     #now we apply reynolds operator to genss
-    if G.group[1] == :SL #TODO other types of reductive groups
+    if group_type(G) == :SL #TODO other types of reductive groups
         new_gens_wrong_ring = [reynolds_function(genss[i], new_rep_mat, new_det, m) for i in 1:length(genss)]
     else 
         return nothing
```

Comment:
Below the fold:
- replace `V = vcat(V,...` by `append!(V, ...`
- `zeros(ringg, n+m^2)` can probably be changed to `zeros(ringg, length(xyz) - n)`
- replace `elemm != 0` by `!is_zero(elemm)`